src/Components/Admin/Rooms/roomModal.js

```diff
@@ -140,38 +149,38 @@ export default function RoomModal({
                         className={classes.textField}
                     />
                 </div>
-                <div>
-                    <FormControlLabel
-                        value="Room is Published"
-                        control={
-                            <Checkbox
-                                onClick={handleIsPublishedClick}
-                                checked={isPublished}
-                            />
-                        }
-                        label="Room is Published"
-                    />
-                    <FormControlLabel
-                        value="Room is Previewable"
-                        control={
-                            <Checkbox
-                                onClick={handleIsPreviewableClick}
-                                checked={isPreviewable}
-                            />
-                        }
-                        label="Room is Previewable"
-                    />
-                </div>
-                <div>
-                    <Typography component="div" variant="h5">
-                        Expected Solve Time:{" "}
-                    </Typography>
-                    <TextField
-                        value={roomSolveTime}
-                        onChange={handleRoomSolveTimeChange}
-                        className={classes.textField}
-                    />
-                </div>
+                {isEdit ? (
```

Comment:
Close! Use `isEdit && (<Component />)` instead of `isEdit ? (<Component />) : null`. The second version is used if you want to render another set of components if the condition is false.

Reply:
Ah good point. Sorry for not mentioning that when we synced Kevin.

Personally, I still use the `<condition> ? <true logic> : null` in case we do need to add false logic in the future, but either is fine.